src/LibpurpleAdapter.cpp

```diff
@@ -301,13 +307,13 @@ guint adapterIOAdd(gint fd, PurpleInputCondition purpleCondition, PurpleInputFun
  * Helper methods
  */
 
-static char* stripResourceFromGtalkUsername(const char* username, const char* serviceName)
+static char* stripResourceFromJabberUsername(const char* username, const char* serviceName)
 {
 	if (!username)
 	{
 		return strdup("");
 	}
-	if (strcmp(serviceName, SERVICENAME_GTALK) != 0)
+	if (strcmp(serviceName, "type_jabber") != 0)
```

Comment:
This one is the only one I kept, because I think it's worth it for the jabber accounts.